suite-native/app/e2e/pageObjects/trading/TradingActions.ts

```diff
@@ -0,0 +1,36 @@
+export class TradingActions {
+    readonly LONG_TIMEOUT = 30000;
+    readonly SHORT_TIMEOUT = 5000;
+    readonly BOTTOM_SHEET_ANIMATION_DURATION = 1000;
+
+    readonly testIdPrefix: string;
+
+    constructor(
+        tradeType:
```

Comment:
nit:
We already use `tradeType`  throughout the trading for different thing and I think we should call it differently here for clarity
```suggestion
        tradeLocationType:
```

Reply:
Updated in f1a3ed4bbfcb1d438e76eed15ec0ece2604695c4.